### ISSUE TITLE
Quicksave load of an Engineer EVA'd with screwdriver leads to out of control spin

### ISSUE THREAD

**Issue by glennfu** [ihsoft/KIS#66]
I've had this happen to me twice using the 1.1.4 release where I loaded a save where an engineer was standing still on the surface. This last time, I EVA'd my engineer, hopped to the ground, then took 1 flexo tube and a screwdriver from the inventory of the pod and transferred it to the inventory of my engineer. Then I hopped close to the module I planned on using it for, then quicksaved.

After that, I did a couple things, messed stuff up, and held F9 to load my last quicksave. Instantly, my engineer is spinning and bouncing out of control, and there's a 5'ish meter long V shaped pole sticking out of her spinning out of control with her. After a few moments, she disappears and my focus is switched to a nearby module. I go to the Space Center, and see something like "Sieselle Kerman could not be recovered... unknownPart".

What does this mean, and what do I do? If I go back to the module, Sieselle Kerman has been placed back into the original module, but now her inventory is empty so the screwdriver is gone.

**Comment by glennfu** [comment]
Some better notes after I re-did the same landing:
1. 2 Kerbals (1 engineer) in an MKS pod.
2. Engineer hops out, can easily hop back in.
3. Engineer grabs a screwdriver, can still hop back in.
4. Engineer grabs 1 Flexotube, CANNOT hop back in. I get an error saying that I can't enter a full pod.

Engineer at this point is permanently bugged. If she puts the 1 Flexotube back in the pod's inventory, she still CANNOT hop back into the pod. If I put her in another pod, she seems to disappear with her portrait missing from the bottom right of the screen. If I switch to the original pod, she's there, but with an empty inventory.

So basically, I can't get a Flexotube from the inventory without bugging out the Engineer.

**Comment by KospY** [comment]
Can you send me your ksp log file when the problem happen ?
Also, what is the Flexotube ? It is part of mks ?

**Comment by glennfu** [comment]
Yes that's part of MKS. I'll work on getting you a log file later today.

**Comment by glennfu** [comment]
So I can't seem to reproduce it today. Maybe it got fixed by the 1.0.2 -> 1.0.4 upgrade? Who knows. However there's still a similar/related issue.

https://dl.dropboxusercontent.com/u/2415512/quicksave.sfs

Go to Minmus Base Alpha, where Seiselle is standing on the ground EVA'd. She has a drill and a flexotube in her inventory. If you get back in the station, then view her inventory, it's empty. EVA again, inventory still empty.

Here's a log file: https://dl.dropboxusercontent.com/u/2415512/KSP.log

**Comment by KospY** [comment]
I think it's related to pod without internal. I heard that MKS has this kind of parts.
This issue should be fixed in the next version of KIS.

**Comment by glennfu** [comment]
Whatever the case, I've been using the same eva + screwdriver + flexotube many time since and haven't hit this bug. I think it's safe to close this!

**Comment by idjlak** [comment]
I have uncontrollable spin with EVA'd engineer, with SC-62 Portable Container on his back. nothing I can do will stop the spin, and it uses up Propellant. Time-warp & "/" stop the spin, but it starts up again immediately.
![image](https://user-images.githubusercontent.com/15899432/70148664-e0325a80-16a6-11ea-854f-aaf89c7c415c.png)

**Comment by ihsoft** [comment]
Well, it's unusual. The container on the back, as well as the other equipped items, are not physical. I.e. they don't have mass or momentum. Could you please recheck this issue in a pristine game, without the other mods?

**Comment by ihsoft** [comment]
And I need the logs of course.
